ci/Readme.md

```diff
@@ -0,0 +1,98 @@
+Instant-CI is a developer tool which aims at providing continous integration to the developers locally on their development systems.
+It runs a container which will be running in the background on the users system, the container will pick up on changes done by the user ( via commits or manual triggers) and has the capability to build install and 
+run PTL tests on the pbs of the users repository, for all this the user may not worry about underlying dependencies or any caveats.
+It also supports build and test history in the form of logs.
+
+Dependincies for this tool are:
+* python3.5 or above
+* docker
+* docker-compose
+
+***How to setup:***
+
+Simply invoke the following command:
+
+` ./ci --start`
+
+***CLI interface for ci:***
+
+* **./ci --start:** This will behave as the setup for ci containers to run in the background. User can provide desired platform ,which must be present in docker hub. Currently supports ubuntu,centos and opensuse.
+```bash
+./ci --start
+ 
+# The above command will run ci with centos:7 as it's base container.
+ 
+./ci --start="ubuntu"
+ 
+# This will run ci with container as "ubuntu", all docker hub images are compatible with ci.
+ 
+./ci --start="opensuse/leap:15" --build
+ 
+# The above command will force the container to rebuild all dependencies.
+```
+
+* **./ci --configure:** user can provide PBS configure options.
+
+```bash
+./ci --configure
+  
+# The above command will run the pbs configure with default options "--prefix=/opt/pbs --enable-ptl" .
+# Note that if you have already specified the configure options and are running it again then 
+# simply calling --configure will run the last called options
+  
+./ci --configure='CFLAGS="-g -O2 -Wall -Werror" --prefix=/tmp/pbs --enable-ptl'
```

Comment:
There's no real reason to use -g.  It's for debugging and we will never be debugging PBS inside the container.